06-building-a-program-start-to-finish/getting-peer-reviews/output/README.md

```diff
@@ -0,0 +1,6 @@
+#### Output
+- **Using**: GitHub
+- **Deliver**: a Pull Request showing at least two Peer Code Reviews
+- **With**: *independently*
+
+[Pull Request with peer reviews](https://github.com/andela-pessien/inverted-index/pull/1)
```

Comment:
Yes I was (most of it) especially the commit message convention and the `max-len` rule